client/plasmacli/sign.go

```diff
@@ -92,31 +83,18 @@ Usage:
 }
 
 // generate confirmation signature for specified owner and position
-func signSingleConfirmSig(ctx context.CLIContext, position plasma.Position, signerAddr, owner ethcmn.Address, name string) error {
-	// retrieve the new output
-	utxo := store.UTXO{}
-	key := append(owner.Bytes(), position.Bytes()...)
-	res, err := ctx.QueryStore(key, "utxo")
+// verify that the inputs provided are correct
+// signing address should match one of the input addresses
+// generate confirmation signature for given utxo
+func signSingleConfirmSig(ctx context.CLIContext, position plasma.Position, signerAddr ethcmn.Address, name string) error {
```

Comment:
use query.TransactionOutput or maybe query.TxOutput, then update formatting below, todo can be taken out.